test/main.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/thedatashed/xlsxreader"
+)
+
+func main() {
+	e, err := xlsxreader.OpenFile("./test.xlsx")
+	if err != nil {
+		fmt.Printf("ERROR OCCURRED: %s", err)
+		return
+	}
+	defer e.Close()
```

Comment:
Is this defer always required?

Reply:
We keep a file descriptor open so data can be streamed from the XLSX file. As a result the file needs to be closed once it's finished with. 

It's up to the calling code to ensure the file is closed, but we need to make sure this is well documented. 

For our use case of server-less, I think some of this may change slightly as:
a) We will most likely hold the xlsx file in memory, and not on disk.
b) Leaving trailing file descriptors is probably not as big an issue, as processes are shorter lived.